modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase

```diff
@@ -245,6 +245,47 @@ definition {
 		}
 	}
 
+	@priority = 4
+	test ExportCustomObjectWithoutObjectEntry {
+		property portal.acceptance = "true";
```

Comment:
Remove from the acceptance as it has priority 3

Reply:
Done, thanks!